.integration/tests/dpc_testnet1.rs

```diff
@@ -54,7 +54,7 @@ fn dpc_testnet1_integration_test() {
             previous_block_hash: BlockHeaderHash([0u8; 32]),
             merkle_root_hash: MerkleRootHash([0u8; 32]),
             pedersen_merkle_root_hash: PedersenMerkleRootHash([0u8; 32]),
-            proof: ProofOfSuccinctWork([0u8; 972]),
+            proof: ProofOfSuccinctWork([0u8; 771]),
```

Comment:
```suggestion
            proof: ProofOfSuccinctWork([0u8; 771]),
```
is this value `PROOF_SIZE` from `ledger`? it would be best if it was imported from there instead

Reply:
Implemented the `Default` trait for `ProofOfSuccinctWork`. 
Replaced with `ProofOfSuccinctWork::default()`.